elements/lisk-p2p/src/utils/select.ts

```diff
@@ -127,9 +127,12 @@ export const selectPeersForConnection = (
 	}
 
 	// LIP004 https://github.com/LiskHQ/lips/blob/master/proposals/lip-0004.md#peer-discovery-and-selection
-	const x =
-		input.triedPeers.length / (input.triedPeers.length + input.newPeers.length);
 	const minimumProbability = 0.5;
+	const x =
+		input.triedPeers.length + input.newPeers.length < 100
```

Comment:
in the LIP it says ` If the collection triedPeers contains less than 100 entries, then a peer is selected uniformly at random from all known peers`

Reply:
Oh yeah, it should be only `triedPeers` < 100